_posts/2022-07-17-scipy_2022.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: SciPy 2022
+---
+
+MDAnalysis recently attended [SciPy 2022](https://www.scipy2022.scipy.org/)!
+
+SciPy was a fantastic opportunity to engage with the open-source
+community, build connections and see the amazing work being conducted all
+across the Scientific Python ecosystem.
+
+Additionally, Hugo MacDermott-Opeskin presented a poster on our ongoing work
+as part of our [CZI-EOSS4 grant]({{ site.baseurl }}{% post_url
+2021-08-31-CZI-EOSS4 %}). You can find his poster at the following
+[FigShare DOI](https://doi.org/10.6084/m9.figshare.20324931.v1).
+
+As part of the SciPy sprints session we ran an MDAnalysis Sprint
+where we invited people to contribute to MDAnalysis and collaborated on
+various ideas with community members. We would like to thank members of the
+[Zarr](https://zarr.dev/), [Xarray](https://docs.xarray.dev/en/stable/),
+[Dask](https://www.dask.org/), and [Pangeo Forge](https://pangeo-forge.org/)
+communities for some helpful and stimulating discussions. 
+
+
+We also participated in a `#bio_at_scipy` meetup to share ideas around engaging
+with the broader community and encouraging contributions.
+
+— The MDAnalysis Team
```

Comment:
... and add @tylerjereddy , too, of course — sorry

It would be nice if there were a core implementation of distance calculations with PBC capabilities in scipy.